interpreter/llvm/src/lib/Transforms/Instrumentation/EfficiencySanitizer.cpp

```diff
@@ -716,7 +716,7 @@ bool EfficiencySanitizer::instrumentGetElementPtr(Instruction *I, Module &M) {
     return false;
   }
   Type *SourceTy = GepInst->getSourceElementType();
-  StructType *StructTy;
+  StructType *StructTy = nullptr;
```

Comment:
LLVM is a third party package. We shouldn't be fixing bugs under `interpreter/llvm/src`. There is a very high chance this issue was already fix and we get it with the next llvm upgrade.